llvm-17.yaml

```diff
@@ -58,6 +58,8 @@ pipeline:
         -DCMAKE_BUILD_TYPE=Release \
         -DCMAKE_INSTALL_PREFIX=${{vars.llvm-prefix}} \
         -DCMAKE_PREFIX_PATH=${{vars.llvm-prefix}} \
+        -DCMAKE_CXX_FLAGS="-U__arm_streaming" \
+        -DCMAKE_C_FLAGS="-U__arm_streaming" \
```

Comment:
Mind adding a comment about this, with a description of the issue and/or a bug link? Something so that a person coming by later will know how to determine if this is still needed.